datasets/naip.yaml

```diff
@@ -0,0 +1,30 @@
+Name: NAIP on AWS
+Description: The National Agriculture Imagery Program (NAIP) acquires aerial imagery during the agricultural growing seasons in the continental U.S. This “leaf-on” imagery andtypically ranges from 60 centimeters to 100 centimeters in resolution. More details [here](https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/).
+Documentation: https://docs.opendata.aws/naip-analytic/readme.html
+Contact: https://www.fsa.usda.gov/programs-and-services/aerial-photography/imagery-programs/naip-imagery/
+UpdateFrequency: NAIP data is delivered at the state level and every year number of states receive updates with an overall update cycle of three years.
+Tags:
+  - aws-pds
+  - earth observation
+  - satellite imagery
```

Comment:
Please remove `satellite imagery` (incorrect) and `multispectral` (I don't think we'll want that as a tag) and add `aerial photography` to https://github.com/awslabs/open-data-registry/blob/master/tags.yaml in this PR.

Reply:
@jflasher 

- Fixed syntax 
- Added the Tag in tags.yaml, used aerial imagery 
- Removed multispectral and satellite imagery